src/main/java/de/qabel/desktop/ui/accounting/AccountingController.java

```diff
@@ -162,4 +168,109 @@ String readFile(File f) throws IOException {
 	ResourceBundle getRessource(){
 		return resourceBundle;
 	}
+
+	Contact gsonContactToContact(GsonContact gc, Identity i) throws URISyntaxException, QblDropInvalidURL {
+
+		ArrayList<DropURL> collection = generateDropURLs(gc.getDropUrls());
+		QblECPublicKey pubKey = new QblECPublicKey(gc.getPublicKey());
+		Contact c = new Contact(i, gc.getAlias(), collection, pubKey);
+		c.setPhone(gc.getPhone());
+		c.setEmail(gc.getEmail());
+
+		return c;
+	}
+
+	Identity gsonIdentityToIdentiy(GsonIdentity gi) throws URISyntaxException, QblDropInvalidURL {
+
+		gi.buildKeyPair();
+		ArrayList<DropURL> collection = generateDropURLs(gi.getDropUrls());
+		QblECKeyPair qblECKeyPair = new QblECKeyPair(gi.getPrivateKey());
+
+		return new Identity(gi.getAlias(), collection, qblECKeyPair);
+	}
+
+	private GsonContact createGsonContact(Identity identity) {
+		GsonContact gc = new GsonContact();
+		gc.setEmail(identity.getEmail());
+		gc.setPhone(identity.getPhone());
+		gc.setAlias(identity.getAlias());
+		gc.setCreated(identity.getCreated());
+		gc.setUpdated(identity.getUpdated());
+		gc.setDeleted(identity.getDeleted());
+		gc.setPublicKey(identity.getEcPublicKey().getKey());
+		for (DropURL d : identity.getDropUrls()) {
+			gc.addDropUrl(d.getUri().toString());
+		}
+		return gc;
+	}
+
+	private GsonIdentity createGsonIdentity(Identity i) {
+		GsonIdentity gi = new GsonIdentity();
+		gi.setEmail(i.getEmail());
+		gi.setPhone(i.getPhone());
+		gi.setAlias(i.getAlias());
+		gi.setCreated(i.getCreated());
+		gi.setUpdated(i.getUpdated());
+		gi.setDeleted(i.getDeleted());
+		gi.setPublicKey(i.getEcPublicKey().getKey());
+		gi.setPrivateKey(i.getPrimaryKeyPair().getPrivateKey());
+		for (DropURL d : i.getDropUrls()) {
+			gi.addDropUrl(d.getUri().toString());
+		}
+		gi.generateKeyStructure();
+		return gi;
+	}
+
+	private void writeJsonInFile(String json, File dir) throws IOException {
+		InputStream stream = new ByteArrayInputStream(json.getBytes(StandardCharsets.UTF_8));
+		byte[] buffer = new byte[stream.available()];
+		stream.read(buffer);
```

Comment:
`json.getBytes` already returns a `byte[]`. Whats the difference after piping it through the stream?